jquery.facetview.js

```diff
@@ -1581,6 +1582,11 @@ search box - the end user will not know they are happening.
                     $(options.searchbox_class).bindWithDelay('keyup',dosearch,options.freetext_submit_delay);
                 }
 
+                // Properly set size for each facet
+                $.each(options.facets, function(facet){
+                  options.facets[facet]["size"] = options.facet_size;
```

Comment:
Yikes, changing the plot output is not good - the plot output should be the only unlimited view of the data, because it is a much more condensed view of a single facet over the time dimension.  A line plot with only 10 data points in it is probably not very interesting.

My comment was simply that you should only set a default value for "size" where "size" is not already explicitly passed.  If it already exists, leave it as-is.

The important distinction to make is the difference between number of facet results to display, per-facet (default of 10 seems sane), and number of results/hits/images/whatever to show.  The latter is the content of the central frame, while the former is the lines of words with (counts) beneath each facet.  I want to change the behavior of the facet results, but I'm less concerned with the other.

Let's also keep an eye on pagination in both cases: we should be able to change an offset, so that second set of 10 results for a facet shows up.